java/arcs/sdk/jvm/CollectionImpl.kt

```diff
@@ -26,27 +26,27 @@ class CollectionImpl<T : Entity>(
     override val size: Int
         get() = entities.size
 
```

Comment:
change to suspend function(size() ), the implementation should be to fetch() and then compute size I think.

Reply:
sounds good to me